src/state/apis/limit-orders/limitOrderApi.ts

```diff
@@ -86,7 +88,23 @@ export const limitOrderApi = createApi({
                   `${baseUrl}/${network}/api/v1/account/${account}/orders?limit=1000`,
                 )
 
-                return result.data.map(order => {
+                // CowSwap limit and spot orders API is the same, so we need to filter out spot orders
+                // there are no parameters to filter from their API, they are filtering after fetching
+                // on their interface as it's some custom metadata they add
+                const limitOrders = result.data.filter(order => {
+                  if (!order.fullAppData) return true
```

Comment:
Can we keep the comment here saying that this shouldn't happen but 🙏🏽

Reply:
Done in [5025d1a](https://github.com/shapeshift/web/pull/9199/commits/5025d1a89c4ca92173fc8b5a7e19801afc9bf8d1)